server/portal/apps/signals/signals.py

```diff
@@ -1,4 +1,7 @@
 from django.dispatch import Signal
 
 # Generic event signal. Not in use as of 6.29.20
-portal_event = Signal(providing_args=['event_type', 'event_data', 'event_users'])
+# portal_event = Signal(providing_args=['event_type', 'event_data', 'event_users'])
```

Comment:
A minor suggestion: Can we delete this instead of commenting if it is not in use. We can go back to file history to look at edits.

Reply:
I removed the commented out code.